main/src/tutorial/tutorial.flix

```diff
@@ -0,0 +1,237 @@
+//
+// Welcome to the introductory Flix Tutorial!
+//
+// In this tutorial we will develop a simple intra-procedural dataflow analysis
+// that performs constant propagation for local variables. The program language
+// we consider is very simple, it has three types of statements:
+//
+//      x = c        [literal statement]
```

Comment:
I was talking to Dan today, and he pointed out that "literal statement" isn't the name he would have used. A literal statement would be just `c`.

What we have (`x = c`) is more like an assignment statement, or literal assignment, if you want to be more specific, I guess.